docs/en/configurations/chaining.md

```diff
@@ -0,0 +1,43 @@
+# Query parameters chaining
+
+> requires ^10.1.0 for Vue 2.x, ^8.6.0 for Vue 1.x
+
+Sometimes you may want to bypass a your custom loader, before pass to `vue-loader`. To do that, you can customize with the `loader` option.
```

Comment:
Let's change this to "you may want to pre-process the vue file before passing it to `vue-loader`"